plugins/experimental/traffic_dump/address.cc

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
I assume the IPAddress classes are not available to the plugins still?

Reply:
Because it's in the core, it is possible to reach around and get `ink_inet.h`.  It might be time to finally push on exposing that to plugins officially.

Reply:
Thanks! IpAddr works perfectly. I wasn't aware it existed.